apps/storybook/src/decorators/withLimitedWidth.tsx

```diff
@@ -0,0 +1,11 @@
+import type { Decorator } from "@storybook/react-vite"
+
+// Function that creates a decorator with a limited width container
+// Provides consistent centering and configurable max-width constraint
+export const withLimitedWidth = (maxWidth: number = 600): Decorator => {
```

Comment:
This will be used in my other PR- not used in this one, but shouldn't hurt to add